### DISCUSSION TITLE
Optionales Erzwingen von Bedingungen zum Steigern

### DISCUSSION THREAD

**Discussion by TomOdenthal** [Plushtoast/dsa5-foundryVTT#1203] [Ideas]
Hallo,
In meiner Spielergruppe geschehen des öfteren Fehler, weil ausversehen Werte gesteigert werden, die laut Regelwerk noch nicht gesteigert werden können weil eine Bedingung nicht erfüllt ist. Genauso werden Steigerungen über direkte Eingabe als Zahl und nicht über +/- umgesetzt, wodurch die AP-Punktzahl nicht mehr stimmt.

Ich verstehe, dass dies eine Designentscheidung ist. Ich würde mir jedoch wünschen, dass man die Einhaltung der Vorgaben des Regelwerks für Helden erzwingen kann. Z.B. als Option in den Einstellungen für Welt oder einzelne Helden.

Die Akzeptanz meiner Spieler Foundry für die Verwaltung von ihren Helden zu nutzen ist leider sehr niedrig, da man ständig aufpassen muss, nichts falsch zu machen und im Prinzip ständig Regeln prüfen muss. 
Ich würde mich sehr freuen, wenn solch ein Feature kommen würde. Idealerweise eine geführte Charaktererstellung/-entwicklung.

**Comment by Plushtoast** [comment]
Die Abhängigkeiten von dsa sind so kompliziert, dass das kaum umsetzbar ist. Wir fokussieren uns auf den spielfluss und da gibt es genug zu tun. Denke eher nicht dass sich hier großartig was ändern wird oder kann

**Reply by TomOdenthal** [reply]
Ok danke für die schnelle Antort. Wäre es denn Denkbar, dass man für Spieler zumindest die "frei Eingabe" von Fertigkeitswerten deaktiviert, sodass nur noch über + und - gesteigert werden kann? Das würde uns schon sehr helfen.
Alternativ einen Neu-Berechnen-Button, mit dem die aktuell ausgegebenen AP ermittelt werden?
Wenn wir die Foundry Charaketerbögen als single source of thruth verwenden, ist es eine große Gefahr, dass irgendwann die AP nicht mehr stimmen.

**Reply by Plushtoast** [reply]
Hier gibt es ein klares nein. Es sind irgendwie 1000 zauber und 4000 sonderfertigkeiten. Die auch nur funktional zu machen ist schon irre genug. Was du aber machen kannst ,wenn du das grundregelwerk modul hast, ist mal /cc oder /help im char einzugeben. Aber selbst das klappt nicht zu 100% weil es viel zu kompliziert ist

**Reply by TomOdenthal** [reply]
Super! Das /cc hilft schon sehr danke :) Guter Hinweis.

Ich verstehe noch nicht ganz, inwiefern das deaktivieren der Textboxen für die Steigerungswerte von der Anzahl oder Funktionalität der Zauber und Sonderfähigkeiten abhängt. Es geht mir nur darum, dass man vermeidet, dass ausversehen Charakteranpassungen durchgeführt werden, die nicht in die AP mit eingerechnet werden.
Der Befehl /cc ist hier aber schonmal eine erste Möglichkeit zur Fehlersuche.

**Comment by mckee14** [comment]
Es war ja von anfang an vorgesehen, in foundry niemals einen vollwertigen
charaktergenerator einzubauen.

Ich unterstütze dies ebenfalls. wieso sollte man das tun? es gibt dafür ja
extra programme.

ich empfehle, den charakter in TheDarkAid zu führen - hat und kann alles.

Cool wäre es, wenn jemand ein import/synchronisierungsmodul schreibt, um
nicht manuell abgleichen zu müssen.

aber ganz ehrlich, einen char zu bauen dauert stunden. ihn dann initial in
foundry anzulegen dauert 15minuten. ihn danach updaten nach steigerungen
dauert... 2 minuten. also sehr überschaubar.

Plushtoast ***@***.***> schrieb am So., 2. Okt. 2022, 14:11:

**Reply by Plushtoast** [reply]
Im chat eingeben nicht char

**Reply by TomOdenthal** [reply]
Was von Anfang an geplant war weiß ich nicht, ist aber auch für die Diskussion an sich nicht relevant.
Als Nutzer von Foundry gehört für mich auch die Charaktererstellung zur ganzheitlichen User-Experience dazu. Wenn Foundry, bzw, das Foundry-DSA-Team sagt, dass für die Charaktererstellung andere tools genutzt werden sollen, ist das schön und gut. Aber dann wäre es sinnvoll zumindest ein tool offiziell zu supporten.
Wenn man auch das nicht tut ist das natrülich auch nicht verwerflich. Aber die Nachfrage eines Nutzers, ob so eine Integration nicht sinnvoll wäre, ist absolut legitim.

Danke für den Hinweis mit TheDarkAid. Kannte ich bisher nicht. Wir haben immer den Optolith genutzt. Da gibt es auch ein Import-Module für Foundry, aber das ist leider nicht 100% kompatibel mit den Foundry-Modulen und sorgt bei uns regelmäßig für Chaos.
Kannst du die Vorteile von TheDarkAid gegenüber Optolith nennen, sofern du beide kennst?

**Comment by mckee14** [comment]
@TomOdenthal Klar ist es legitim für eine Diskussion. Ich führte das nur detaillierter aus, weil es von Anfang an so definiert wurde und Plushtoast das auch bereits dutzendfach erklärt hat. Natürlich wissen neue Spieler davon nichts und solche Nachfragen sind nur natürlich.

Zu DarkAid und Optilith:

- Meines Wissens ist der "Selbsrechnende Heldenbogen" als PDF offiziell von Ulisses? Das ist natürlich nicht sehr intuitiv, performant und automatisiert. Aber sehr komplett und in der Berechnung der AP korrekt. Falls das Ding NICHT offiziell ist, dann gehe ich seit Jahren fälschlicherweise davon aus :)
  - Dieses Tool nutze ich immer noch, wenn ich einen komplexen Char gegenprüfen möchte bezüglich AP.
- Optolith
  - Eingängige Nutzerführung, simplifiziert
  - Du bekommst nur zu sehen, was Du auch kaufen/wählen darfst, sprich: wofür Du die Voraussetzungen erfüllst
  - Meines Wissens nicht komplett in den Inhalten. Wobei ich seit TDA Nutzung nicht mehr regelmässig nachschaue.
 - TheDarkAid
   - Sehr viele Optionen / Optionalregeln
   - Sehr Umfangreich
   - Zeigt alle (aus den aktivierten Regelwerken) Inhalte an. Zeigt, welche direkt verfügbar sind und welche noch Voraussetzungen benötigen. Diese lassen sich auch direkt dazukaufen (Bsp. Finte I -> Du siehst ein Ausrufezeichen und klickst drauf -> Da steht, dass Du einen zu tiefen Eigenschaftswert hast -> Klick, du steigerst die Eigenschaft auf die Voraussetzung ohne hin und her springen)
   - Weniger eingängige GUI
   - Gibt das Charakterblatt anders formatiert aus, als alle anderen Charakterblätter.

in TDA muss man sich einmalig 10min reinfuchsen, danach ist es imho mit abstand der beste Generator für DSA5.

Begleittiere können sie alle nicht gut. Für maximal 1 Stk. ist der Selbstrechnende Heldenbogen noch der beste.

**Reply by TomOdenthal** [reply]
Danke für die Ausführliche Erläuterung! :)
Ich werde mit TDA mal anschauen

**Comment by Plushtoast** [comment]
Was man alles "professionell" machen könnte is auch schön und gut, ändert aber nix ander Tatsache, dass das hier nur ein Nebenberufliches Projekt für eine Nische ist  und irgendwo dann auch die Zeit begrenzt ist.

**Reply by TomOdenthal** [reply]
Ich muss zugeben, dass ich dachte, das Projekt wäre offiziell durch Ulisses finanziert. Wenn das ein privates Projekt ist, dann echt mal Hut ab. Gute Arbeit!

**Comment by Plushtoast** [comment]
Ist es nicht. 
Hier die Erklärung wie das so läuft.
https://www.youtube.com/watch?v=royjo135P2Q&ab_channel=UlissesSpiele
